ladybug/src/main/resources/springIbisDebuggerAdvice.xml

```diff
@@ -123,17 +123,18 @@
 			<aop:around
 				pointcut=
 					"
-	 					execution(
-	 						*
-	 						nl.nn.adapterframework.stream.IOutputStreamingSupport.provideOutputStream(
-	 							nl.nn.adapterframework.core.IPipeLineSession, nl.nn.adapterframework.stream.IOutputStreamingSupport
-	 						)
+	 				execution(
+	 					*
+	 					nl.nn.adapterframework.stream.IOutputStreamingSupport.provideOutputStream(
+	 						nl.nn.adapterframework.core.IPipeLineSession, 
+	 						..
```

Comment:
Wat doet dit? .. ?

Reply:
Dit verwijst naar optionele extra argumenten. Dus met of zonder IForwardTarget, in dit geval